api/src/main/java/org/openmrs/Order.java

```diff
@@ -32,10 +32,18 @@
  * 
  * @version 1.0
  */
-public class Order extends BaseOpenmrsData implements java.io.Serializable {
+public class Order extends BaseOpenmrsData implements java.io.Serializable, Comparable{
 	
 	public static final long serialVersionUID = 4334343L;
-	
+
+	@Override
+	public int compareTo(Object o) {
```

Comment:
First point understood.

Second Point: Yes the compare I will delegate to the Integer class.
But even then I will need line 44 for the else clause right?
Because the compare logic is only for orders that are having sequenceNumber set for them,
not for others.
So I would anyway need the else return logic.
Am I understanding correctly?